src/main/java/org/opensearch/security/action/onbehalf/CreateOnBehalfOfTokenAction.java

```diff
@@ -164,7 +151,7 @@ public void accept(RestChannel channel) throws Exception {
                         tokenDuration,
                         mappedRoles.stream().collect(Collectors.toList()),
                         user.getRoles().stream().collect(Collectors.toList()),
-                        roleSecurityMode
+                        false
```

Comment:
If im understanding this correct, we need this into `true` to have encrypted role claims.

Reply:
I've changed the parameter to be called `includeBackendRoles`, this inverted the previous (IMO confusing) true/false statements.  With a value of `false` the backend roles will not be included in the JWT.